utils/wasm-instrument/src/lib.rs

```diff
@@ -237,7 +238,9 @@ pub fn inject<R: Rules>(
         None => return Err(mbuilder.build()),
     };
 
-    let cost = cost_push_arg + cost_call + cost_blocks;
+    let cost_local_var = rules.call_per_local_cost() as u64;
+
+    let cost = cost_push_arg + cost_call + cost_local_var + cost_blocks;
```

Comment:
can we have a proof that it become cheaper?

Reply:
That's really nice performance improvement!

Reply:
What do you mean by "how does this affect the fungible token"? This cost will be the same for all contracts. Because the `gas_charge` function is inserted at the end and always consists of the same instructions

Reply:
> What do you mean by "how does this affect the fungible token"? This cost will be the same for all contracts. Because the `gas_charge` function is inserted at the end and always consists of the same instructions

Sure, but what's wrong with some real example? Gas costs will differ between contract with 123012030 calls of gas charge and for example 50 :/

Reply:
@breathx I am using `fungible-token::stress_transfer`. It shows the following results:
```
# this PR
Gas burned for one transfer operation = 1.3684201959330864 * 10^9. Calculated as geometric mean from 100 transfer operations.

# master
Gas burned for one transfer operation = 1.3721279099998007 * 10^9. Calculated as geometric mean from 100 transfer operations.
```

How it was tested:
```bash
cargo build --package gear-cli --release
cargo test --package demo-fungible-token --release stress_transfer -- --exact --include-ignored --nocapture
```

```diff
diff --git a/examples/binaries/fungible-token/tests/benchmarks.rs b/examples/binaries/fungible-token/tests/benchmarks.rs
index 10a255d8f..2ce34ad1d 100644
--- a/examples/binaries/fungible-token/tests/benchmarks.rs
+++ b/examples/binaries/fungible-token/tests/benchmarks.rs
@@ -23,7 +23,7 @@ use ft_io::*;
 use gclient::{EventProcessor, GearApi, Result};
 use gear_core::ids::{MessageId, ProgramId};
 use gstd::{vec, ActorId, Encode, Vec};
-use rand::Rng;
+use rand::{rngs::StdRng, Rng, SeedableRng};
 use statrs::statistics::Statistics;

 /// Path to the gear node binary.
@@ -206,7 +206,7 @@ async fn stress_test() -> Result<()> {
 #[ignore]
 #[tokio::test]
 async fn stress_transfer() -> Result<()> {
-    let mut rng = rand::thread_rng();
+    let mut rng = StdRng::seed_from_u64(0xdeadbeef);

     let api = GearApi::dev_from_path(GEAR_PATH).await?;
     // Use this code in comment for custom node run:
```